app/src/common/shared/com/igalia/wolvic/browser/SettingsStore.java

```diff
@@ -85,11 +85,11 @@ SettingsStore getInstance(final @NonNull Context aContext) {
     public final static int UA_MODE_DEFAULT = WSessionSettings.USER_AGENT_MODE_VR;
     public final static int INPUT_MODE_DEFAULT = 1;
     public final static float DISPLAY_DENSITY_DEFAULT = 1.0f;
-    public final static int WINDOW_WIDTH_DEFAULT = 800;
-    public final static int WINDOW_HEIGHT_DEFAULT = 450;
+    public final static int WINDOW_WIDTH_DEFAULT = 1024;
```

Comment:
I wonder whether we should rename these to WINDOW_TEX_[WIDTH|HEIGHT]_DEFAULT so we don't have to review everytime which width|height are we talking about.